packages/perseus-editor/package.json

```diff
@@ -43,6 +43,7 @@
         "@khanacademy/perseus-core": "workspace:*",
         "@khanacademy/perseus-score": "workspace:*",
         "@khanacademy/pure-markdown": "workspace:*",
+        "katex": "0.11.1",
```

Comment:
I've moved `katex` from `devDependencies` to `dependencies`.

KaTeX is needed by perseus-editor, to display TeX parsing errors. Unfortunately, MathJax does not produce very useful syntax error messages, so we have to use KaTeX for this.

This was working before (with KaTeX in devDependencies) because webapp depended on KaTeX, so it was available when the editor imported it.